main/nav_backplane_viewer.py

```
@@ -0,0 +1,1366 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+from typing import Any, Callable, Optional, cast
```

Ruff (0.14.5)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)